bundles/binding/org.openhab.binding.dscalarm/META-INF/MANIFEST.MF

```diff
@@ -4,7 +4,7 @@ Bundle-SymbolicName: org.openhab.binding.dscalarm
 Bundle-Version: 1.8.0.qualifier
 Bundle-Activator: org.openhab.binding.dscalarm.internal.DSCAlarmActivator
 Bundle-Vendor: openHAB.org
-Bundle-RequiredExecutionEnvironment: J2SE-1.5
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6
```

Comment:
I was getting a "must override a superclass method" error after adding the new method to the binding, so I bumped the execution environment up to 1.6.